src/plugins/discover/public/index.ts

```diff
@@ -28,6 +27,13 @@ export type {
   UnifiedHistogramCustomization,
   TopNavCustomization,
 } from './customizations';
-export { SEARCH_EMBEDDABLE_TYPE, SEARCH_EMBEDDABLE_CELL_ACTIONS_TRIGGER_ID } from './embeddable';
-export { loadSharingDataHelpers } from './utils';
+export {
+  SEARCH_EMBEDDABLE_TYPE,
+  SEARCH_EMBEDDABLE_CELL_ACTIONS_TRIGGER_ID,
+  apiPublishesSavedSearch,
+  type PublishesSavedSearch,
+  type HasTimeRange,
+  type SearchEmbeddableSerializedState,
+} from './embeddable';
+export { loadSharingDataHelpers, getSortForEmbeddable } from './utils';
```

Comment:
nit, does `getSortForEmbeddable` need to be exported? I don't see it being used outside of this plugin?

Reply:
Good catch! I think this was leftover from when I was attempting to move the embeddable from the `discover` plugin to the `savedSearch` (gave up on that as part of this effort 🙈). Fixed in ff011070b05fb042fff709122bb51b862fbdb68f